source/customizations/profiles.inc

```diff
@@ -0,0 +1,84 @@
+.. _profiles_guide:
+
+Configuration Profiles
+----------------------
+
+.. note::
+
+   The profile based configuration is an experimental feature. Not all properties have been migrated to be compatible.
+   See the list the list of supported properties in the :ref:`configuration documentation <profile_properties>`
+
+Configuration properties can be organized in a hierarchy using profiles. With profiles,
+administrators can create different configuration settings within the same OnDemand installation.
+A profile can be pre-selected or dynamically changed while using the Dashboard application.
+
+To define profile properties, we have group them under a profile name within the ``profiles`` keyword.
+Multiple profiles can be created using different files, the configuration
+logic will aggregate all profiles into a single consolidated object.
+
+.. code-block:: yaml
+
+  profiles:
+    profile_name:
+      config_property: config_value
+
+OnDemand uses an inheritance and override approach for profiles.
+The root configuration provides default properties that all profiles inherit.
+Then, profile properties can override these values or define new ones as needed.
+
+The example below shows how the root profile defines a value for ``pinned_apps`` and ``pinned_apps_menu_length``.
+The ``rstudio_group`` profile inherits the property value for ``pinned_apps_menu_length``,
+and overrides the value for ``pinned_apps`` just to the RStudio application.
+As well, it defines the property ``pinned_apps_group_by``.
+
+.. code-block:: yaml
+
+  pinned_apps: [sys/*]
+  pinned_apps_menu_length: 10
+
+  profiles:
+    rstudio_group:
+      pinned_apps: [sys/rstudio]
+      pinned_apps_group_by: "department"
+
+.. _profiles_selection_guide:
+
+Manual Profile Selection
+........................
+
+The selection of a profile can be done manually by the user or automatically based on request data.
+
+Links can be added the Help naviagtion menu to dynamically switch between profiles.
+These links can be configured using the configuration property ``link_profiles``.
+
+.. code-block:: yaml
+
+  profile_links:
+    - id: ""
+      name: "Root Profile"
+      icon: "cog"
```

Comment:
Isn't this `icon_uri`?

Reply:
No it is `icon`, we use `icon_uri` in the other places...

Reply:
I notice this while doing the documentation.
I am working on a small PR to make this consistent with the NavBar configuration.
After the change, the config above will look like this:

```yaml
  profile_links:
    - title: "Root Profile"
      profile: ""
      icon_uri: "fas://cog"
    - title: "Rstudio Profile"
      profile: rstudio_group
      icon_uri: "fas://user"
    - title: "Jupiter Profile"
      profile: jupyter_group
      icon_uri: "fas://user"
```
Will update the documentation when the PR is merged

Reply:
I am wondering if we should just use `icon` everwhere and force `fas`. Or at least interpret the string and see if it's a full URI. if so, we do nothing, if not, then we default to `fas`.

Reply:
everywhere that's safe to do it.  We'd have to consider compatibility when modifying `manifest` and `batch_connect` forms, so we can defer that or tackle it as a separate thing entirely.

In any case, I would +1 we launch the new navigation with just `icon` that can infer scheme and default to fas.